src/main/vectormenu.ts

```diff
@@ -0,0 +1,188 @@
+import { app, shell, Menu, MenuItem, MenuItemConstructorOptions } from 'electron';
+
+import { ENVIRONMENT, PLATFORM } from '@shared/constants';
+
+export function buildMenuTemplate(): Menu {
+  const template: Array<MenuItemConstructorOptions | MenuItem> = [
```

Comment:
```suggestion
  const Template: MenuItemConstructorOptions[] | MenuItem[] = [
```

Reply:
Agree with the type, but why the constant name starts from the capital `T`?